client/src/js/components/syllabus/CourseItem.js

```diff
@@ -334,64 +649,47 @@ const CourseItem = ({
                 transform="shrink-2"
               />
             </a>
-            <InvisibleButton
+            <InvisibleButton /* Add Button */
               onClick={(e) => {
                 e.preventDefault();
                 handleOnClick(title, lng);
               }}
             >
               {buttonIcon}
             </InvisibleButton>
-            <MediaQuery maxWidth={sizes.desktop}>
-              {(matches) => {
-                return (
-                  matches &&
-                  !isInCourseEvalsPage &&
-                  course.e && (
-                    <ViewEvalsIconButton
-                      href={`/courseEvals?courseID=${syllabusId}&searchLang=${searchLang}`}
-                    >
-                      <FontAwesomeIcon icon={faCommentDots} size="2x" />{" "}
-                    </ViewEvalsIconButton>
-                  )
-                );
-              }}
-            </MediaQuery>
+
+            {reviewButtonIcon}
+            {shareButtonIcon}
```

Comment:
I can see the SNS buttons only in mobile mode. It just shows one gray share button in PC mode.
I saw you use `<MediaQuery maxWidth={sizes.desktop}>` , which display the contained components only in mobile mode (the screen is narrower than a normal PC screen). If you want to display the button in the same style and in PC mode, don't use MediaQuery. If you want to display a different style in PC mode, you have to use one more MediaQuery like `<MediaQuery minWidth={sizes.desktop}>` .

Also, I suggest you to create a new component named sth like `ShareButtonsContainer` in a new JS file, and then move the share button and these SNS button under that component. The code is easier to read and manage in this way. You don't need to do it right now if you feel more convenient to develop in the current way.